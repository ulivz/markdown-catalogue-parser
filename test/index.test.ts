import fs from 'fs'
import path from 'path'
import markdownCatalogueParser from '../src/index'

test('main', () => {
  const rawMarkdown: string = <string>fs.readFileSync(path.resolve(__dirname, '../README.md'), 'utf8')
  const catelogue: string = markdownCatalogueParser(rawMarkdown)
  expect(catelogue).toMatchSnapshot()
})
