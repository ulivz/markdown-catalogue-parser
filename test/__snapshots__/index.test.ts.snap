// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `
"  - [markdown-catalogue-parser](#markdown-catalogue-parser)
    * [Install](#install)
    * [Usage](#usage)
    * [Contributing](#contributing)
    * [Author](#author)"
`;
